protoc_docs/code.py

```diff
@@ -103,4 +103,4 @@ def get_python_docstring(self):
             answer += '    %s:\n%s\n' %  (k, '\n'.join(tw.wrap(v)))
 
         # Done.
-        return answer
+        return answer.encode('utf-8')
```

Comment:
This makes the return value a `bytes`, not a string? Is that intentional? It seems like a strange decision.

Reply:
TBH I used this change locally while running this plugin and it didn't change the diffs on the doc strings, so I rolled with it. But I am happy to change it to whatever you think is best.

Reply:
Were you running on Python 2.7 or Python 3?